src/pages/Admin/MapPreview/MapPreview.tsx

```diff
@@ -171,12 +174,13 @@ const MapPreview: React.FC<MapPreviewProps> = ({
               height: 100,
             }}
             otherIconsStyle={{ opacity: 0.4 }}
-            // lockAspectRatio
+            lockAspectRatio
```

Comment:
[OOS] Huh.. neat that this already existed!

That means that it would be fairly trivial (in the future) to implement a concept of being able to change the aspect ratio when holding a 'modifier key', as described in https://github.com/sparkletown/internal-sparkle-issues/issues/371#issuecomment-778673339

Reply:
(Might be worth creating a new issue to backlog this part/suggestion?)

Reply:
Backlogged in https://github.com/sparkletown/internal-sparkle-issues/issues/392